integration/index.html

```diff
@@ -0,0 +1,117 @@
+        </section>
+
+        <section class="etudes">
+            <h2>ETUDES</h2>
+
+            <article class="personal info">
+                <img src="https://via.placeholder.com/150" alt="picture_for_portolio">
+                <ul>
+                    <li>Lagrabe</li>
+                    <li>Matthias</li>
+                    <li>19</li>
+                    <li>matthias.lagrabe@gmail.com</li>
+                    <li>0637580018</li>
+                </ul>
+            </article>
+            
+            <article class="graduate studies">
+                <ol>
+                    <li>DATE <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam voluptas dolor corrupti obcaecati dolorem atque aliquid est. Eos perferendis debitis laborum nam a, molestiae magnam repellat? Repellendus sunt minus natus.</p></li>
+                    <li>DATE <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam voluptas dolor corrupti obcaecati dolorem atque aliquid est. Eos perferendis debitis laborum nam a, molestiae magnam repellat? Repellendus sunt minus natus.</p></li>
+                    <li>DATE <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam voluptas dolor corrupti obcaecati dolorem atque aliquid est. Eos perferendis debitis laborum nam a, molestiae magnam repellat? Repellendus sunt minus natus.</p></li>
+                    <li>DATE <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam voluptas dolor corrupti obcaecati dolorem atque aliquid est. Eos perferendis debitis laborum nam a, molestiae magnam repellat? Repellendus sunt minus natus.</p></li>
+                </ol>
+            </article>
+        </section>
+
+        <section class="articles">
+            <h2>Articles</h2>
+
+            <article class="article__title">  
+                <img src="https://via.placeholder.com/300x200" alt="article1"/>
+                <h3 class="article__title">Article 1</h3>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel ab debitis eveniet ipsum illo voluptatibus eos, facilis omnis enim autem, iste, cumque error quos placeat tempore repudiandae minus hic et!</p>
+                <button>Lire</button>
+                <button>X commentaires</button>
+            </article>
+
+            <article class="article__title">  
+                <img src="https://via.placeholder.com/300x200" alt="article2"/>
+                <h3 class="article__title">Article 2</h3>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel ab debitis eveniet ipsum illo voluptatibus eos, facilis omnis enim autem, iste, cumque error quos placeat tempore repudiandae minus hic et!</p>
+                <button>Lire</button>
+                <button>X commentaires</button>
+            </article>
+        </section>
+
+        <section class="contact">
+            <h2>Contact</h2>
+            
+            <form>
+                <label>Nom et Prénom
```

Comment:
<'label'> Nom et prénom </'label'> puis input
manque l'attribut for dans le label